arangosh/Dump/DumpFeature.cpp

```diff
@@ -222,8 +222,12 @@ bool isIgnoredHiddenEnterpriseCollection(arangodb::DumpFeature::Options const& o
                                          std::string const& name) {
 #ifdef USE_ENTERPRISE
   if (!options.force && name[0] == '_') {
-    if (strncmp(name.c_str(), "_local_", 7) == 0 ||
-        strncmp(name.c_str(), "_from_", 6) == 0 || strncmp(name.c_str(), "_to_", 4) == 0) {
+    if (strncmp(name.c_str(), arangodb::StaticStrings::FullLocalPrefix.c_str(),
```

Comment:
I have seen this method now multiple times. Can we unify it and move it into EE code?

Reply:
Tried to. Struggled with includes of our dumpfeature. As discussed on slack - skip this for now.